cgi-bin/LJ/User/Account.pm

```diff
@@ -822,6 +822,8 @@ sub preload_props {
                 LJ::get_db_reader();
             $used_slave = 1;
         }
+        die "No db\n" unless $db;
+
```

Comment:
This should probably either give a bit more context, or remove the "\n" so that the line number gets printed.